agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -264,20 +264,24 @@ async def create_image(
     docker_id: str,
     tags: str,
     user: UserDB,
+    template_image: bool,
```

Comment:
we should directly pass deletable. The db_manager should not implement business logic. It should only work as an interfact to the db.

Reply:
Also, please make sure to update all the calls to this function with the new params